oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/MetadataRegistry.java

```diff
@@ -278,7 +278,7 @@ Duration downSamplingDuration(DownSampling downSampling) {
             case Minute:
                 return Duration.ofMinutes(1);
             case Day:
-                return Duration.ofDays(1);
+                return Duration.ofHours(24);
```

Comment:
FYI @apache/skywalking-committers This is overriding a BanyanDB client bug AFAIK.

When Client 0.7 is out, it doesn't matter it is `ofDays(1)` or this.

As this is a serious bug, I will go for a hot fix(10.0.1) just add this one.

Reply:
apache/skywalking-banyandb-java-client#58 fixes the client's bug